plugins/polio/models.py

```diff
@@ -1054,6 +1054,65 @@ class VaccineArrivalReport(SoftDeletableModel):
     objects = DefaultSoftDeletableManager()
 
 
+class VaccineStock(models.Model):
+    account = models.ForeignKey("iaso.account", on_delete=models.CASCADE, related_name="vaccine_stocks")
+    country = models.ForeignKey(
+        "iaso.orgunit",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="vaccine_stocks",
+        help_text="Unique (Country, Vaccine) pair",
+    )
+    vaccine = models.CharField(max_length=5, choices=VACCINES)
+
+    class Meta:
+        unique_together = ("country", "vaccine")
+
+
+# Form A
+class OutgoingStockMovement(models.Model):
+    campaign = models.ForeignKey(Campaign, on_delete=models.CASCADE)
+    vaccine_stock = models.ForeignKey(
+        VaccineStock, on_delete=models.CASCADE
+    )  # Country can be deduced from the campaign
+    report_date = models.DateField()
+    form_a_reception_date = models.DateField()
+    usable_vials_used = models.PositiveIntegerField()
+    unusable_vials = models.PositiveIntegerField()
+    lot_numbers = ArrayField(models.CharField(max_length=200, blank=True), default=list)
+    missing_vials = models.PositiveIntegerField()
+
+
+class DestructionReport(models.Model):
+    vaccine_stock = models.ForeignKey(VaccineStock, on_delete=models.CASCADE)
+    action = models.TextField()
```

Comment:
I put a comment in the mock up but I think we need to clarify this one. If we leave it as a text field we won't be able to translate it, and from the mockup it's only "Destruction + district"